src/test/java/io/github/ust/mico/kafkafaasconnector/TestUtils.java

```diff
@@ -0,0 +1,51 @@
+package io.github.ust.mico.kafkafaasconnector;
+
+import io.github.ust.mico.kafkafaasconnector.configuration.KafkaConfig;
+import org.springframework.kafka.test.EmbeddedKafkaBroker;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+
+public class TestUtils {
```

Comment:
Maybe merge this with kafka test utils as the methods in this class also have a kafka theme.